src/test/resources/application.properties

```diff
@@ -3,3 +3,10 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+spring.jpa.open-in-view=false
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.show-sql=true
+
+jwt.expiration=600000
+jwt.secret=ILoveGreenAndBigShrekFromTheSwamp
```

Comment:
It’s  funny))  but it’s a project for your CV  so better use more secure` jwt.secret`